wikiconv/ingest_revisions/BUILD

```diff
@@ -1,5 +1,27 @@
 py_binary(
     name = "dataflow_main",
     srcs = ["dataflow_main.py"],
-    deps = ["//ingest_utils:wikipedia_revisions_ingester"],
+    deps = ["//wikiconv/ingest_revisions/ingest_utils:wikipedia_revisions_ingester"],
+)
+
+# Requires cloud project auth,
+# bazel run :dataflow_tester
+py_binary(
+    name = "dataflow_tester",
+    srcs = ["dataflow_tester.py"],
+    data = ["testdata/test_wiki_dump.xml.bz2"],
+    deps = [
+        ":dataflow_main",
+        "@bazel_tools//tools/python/runfiles",
```

Comment:
nit: does this require two slashes (`//`) after `@bazel_tools`?

Reply:
Yes, required.